reinforcement_learning/bindings/cs/rl.net/LiveModel.cs

```diff
@@ -22,13 +22,35 @@ private static New<LiveModel> BindConstructorArguments(Configuration config)
         private static extern int LiveModelInit(IntPtr liveModel, IntPtr apiStatus);
 
         [DllImport("rl.net.native.dll")]
-        private static extern int LiveModelChooseRank(IntPtr liveModel,  [MarshalAs(NativeMethods.StringMarshalling)] string eventId,  [MarshalAs(NativeMethods.StringMarshalling)] string contextJson, IntPtr rankingResponse, IntPtr apiStatus);
+        private static extern int LiveModelChooseRank(IntPtr liveModel, [MarshalAs(NativeMethods.StringMarshalling)] string eventId, [MarshalAs(NativeMethods.StringMarshalling)] string contextJson, IntPtr rankingResponse, IntPtr apiStatus);
 
         [DllImport("rl.net.native.dll")]
-        private static extern int LiveModelReportOutcome(IntPtr liveModel,  [MarshalAs(NativeMethods.StringMarshalling)] string eventId,  float outcome, IntPtr apiStatus);
+        private static extern int LiveModelReportOutcomeF(IntPtr liveModel, [MarshalAs(NativeMethods.StringMarshalling)] string eventId, float outcome, IntPtr apiStatus);
```

Comment:
Can't we have the same name with different signatures ?

Reply:
We cannot, because we have to have this function be treated as a c-style function (to avoid name mangling, so P/Invoke can find it), and c does not allow function overloading, so if we try to compile the rl.net.native project with two exports of LiveModelReportOutcome we get compile errors.